api/src/db/DynamoUserDataClient.ts

```diff
@@ -141,6 +119,31 @@ export const DynamoUserDataClient = (
     return await search(statement);
   };
 
+  const getUsersWithBusinesses = async (
+    ExclusiveStartKey?: any
```

Comment:
question, is this type a string?

Reply:
This is essentially the same as LastEvaluatedKey . Both refer to the mechanism used by DynamoDB for paginating query results and is not a string.